content/docs/2.7/scalers/gcp-storage.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Google Cloud Platform Storage"
+availability = "2.7+"
+maintainer = "Community"
+description = "Scale applications based on the count of objects in a given GCP Storage bucket."
+layout = "scaler"
+go_file = "gcp_storage_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `gcp-storage` scaler, which scales Kubernetes workloads based on the count of objects in a given Google Cloud Storage (GCS) bucket. This scaler assumes the worker, when run, will process and clear the bucket by deleting/moving objects therein.
+
+```yaml
+triggers:
+- type: gcp-storage
+  metadata:
+    bucketName: test-bucket
+    targetObjectCount: '100'
+    maxBucketItemsToScan: '1000'
+    credentialsFromEnv: GOOGLE_APPLICATION_CREDENTIALS_JSON # Optional
+    credentialsFromEnvFile: GOOGLE_APPLICATION_CREDENTIALS_JSON # Optional
+```
+
+**Parameter list:**
+
+- `bucketName` - Name of the bucket in GCP Storage.
```

Comment:
now that you've called it GCS above, I think this and the title (above) should change to GCS as well

```suggestion
- `bucketName` - Name of the bucket in GCS.
```